db/migrate/20210720141304_create_municipe.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateMunicipe < ActiveRecord::Migration[6.1]
+  def change
+    create_table :municipes do |t|
+      t.string :name
+      t.integer :cpf
```

Comment:
Refactor: Por aqui eu já sei que sua validação uniqueness não irá funcionar. Afinal de contas, quem garante a validação dos dados, é o banco de dados ou o ORM (activerecord) ?

Leitura RECOMENDADA: https://thoughtbot.com/blog/the-perils-of-uniqueness-validations